tests/all.js

```diff
@@ -0,0 +1,6 @@
+define([
```

Comment:
Since we only have two suites, it seems like it'd be simpler to just have two lines in the config's `suites` parameter rather than adding a new module.